src/components/PointEditor/index.js

```diff
@@ -143,7 +143,7 @@ const PointEditor = ({ isEdit }) => {
       <form className={pointEditor} onSubmit={handleSubmit}>
         <div className={pointEditorHeader}>
           <h4>{isEdit ? 'Edit Data' : 'Add Data to Record'}</h4>
-          <button className={closeAction} onClick={handleClose}>
+          <button id="close" className={closeAction} onClick={handleClose}>
```

Comment:
Something like `id="point-editor-close"` would make the selector a little more unique, thus limiting any chance of duplicate IDs in the document, but it's not a big deal since we have full control over the output markup.